.vortex/CLAUDE.md

```
@@ -165,6 +195,40 @@ info "Finished executing example operations in non-production environment."
 2. **Update BATS Tests**: Update test assertions in `.vortex/tests/bats/`
 3. **Update Installer Fixtures**: Use `UPDATE_FIXTURES=1` process
 
+### Provision Script BATS Test Logic
+
+The provision example script (`scripts/custom/provision-10-example.sh`) uses conditional logic:
+```bash
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines
null

(MD031, blanks-around-fences)